docroot/sites/all/themes/custom/wsdot_classic/templates/page--404.tpl.php

```diff
@@ -1,105 +1,123 @@
-<script>
-  $(window).on('load resize',function(){
-    $wheel = $('#wheel');
-    var offset = $wheel.offset();
-
-    function mouse(e){
-      var center_x = (offset.left)+($wheel.width()/2);
-      var center_y = (offset.top)+($wheel.height()/2);
-      var mouse_x = e.pageX;
-      var mouse_y = e.pageY;
-
-      var radians = Math.atan2(mouse_x-center_x,mouse_y-center_y);
-      var degree = (radians*(180/Math.PI)*-1)+180; 
-
-      $wheel.css({
-        "-moz-transform":"rotate("+degree+"deg)",
-        "-webkit-transform":"rotate("+degree+"deg)",
-        "-o-transform":"rotate("+degree+"deg)",
-        "-ms-transform":"rotate("+degree+"deg)",
-        "transform":"rotate("+degree+"deg)"
-      });
-    }
-
-    $(document).mousemove(mouse);
-  });
-</script>
+<a href="#skipnav" class="skip">Skip Top Navigation</a> <a href="#skiptocontent" class="skip">Skip to Content</a> <a name="top"></a>
+<div id="header">
+  <div class="topban">
+    <ul class="topnav">
+      <li><a href="/">WSDOT Home</a></li>
+      <li><a href="/contact">Contact Us</a></li>
+      <li><a href="/goodtogo">Good To Go!</a></li>
+      <li><a href="/employment">Employment</a></li>
+      <li class="first"><a href="/news">News</a></li>
+    </ul>
+  </div>
+  <div class="logo">
+    <h1> <a href="/"><img src="/sites/all/themes/custom/wsdot_classic/images/blacklogo.gif" alt="WSDOT" /></a> </h1>
+    <div class="emailupdates"><a href="https://public.govdelivery.com/accounts/WADOT/subscriber/new" onclick="window.open('https://public.govdelivery.com/accounts/WADOT/subscriber/new','Popup','width=780,height=440,toolbar=no,scrollbars=yes,resizable=yes'); return false" title="Sign up for email updates">Email updates</a></div>
+  </div>
+
+  <div class="mainban">
+    <div class="mainnav">
+
+      <ul>
+        <li><a href="/traffic" title="Statewide Traveler Information">Traffic &amp; Cameras</a></li>
+        <li><a href="/projects" title="Highway, Ferry and Rail Construction Projects">Projects</a></li>
+        <li><a href="/business" title="Information on Doing Business with WSDOT">Business</a></li>
+        <li><a href="/environment" title="What does WSDOT do for the environment?">Environment</a></li>
+        <li><a href="/mapsdata.htm" title="Maps, Publications and Transportation Data">Maps &amp; Data</a></li>
+      </ul>
+
+      <div class="search">
+        <form id="searchform" method="get" action="https://www.wsdot.wa.gov/search/">
+          <fieldset>
+            <input type="text" name="q" id="query-field" class="search-text" />
+            <input type="submit" value="Search" class="searchButton" />
+          </fieldset>
+        </form>
+      </div>
+    </div>
+  </div>
+</div>
+
+<div id="wrapper">
+  <div id="main" class="main-404">
+    <img src="https://www.wsdot.wa.gov/media/images/4.png">
+    <img id="wheel">
+    <img src="https://www.wsdot.wa.gov/media/images/4.png">
+    <div id="section">
```

Comment:
The above two img src attributes need to be updated to ".../sites/all/themes/custom/wsdot_classic/images/wheels/" as well.

Reply:
Never mind. I got it.